### DISCUSSION TITLE
Summary page

### DISCUSSION THREAD

**Discussion by nicoloceneda** [ImperialCollegeLondon/rse_introductory_git_course#31] [Ideas]
The content of the course is excellent. Yet, since it is highly descriptive (as it should be), it is sometimes time consuming to go back and find exactly what you are looking for. It would be very nice to add a page that summarises all key commands and cases in a concise way.

**Comment by alexdewar** [comment]
Just noticed this old discussion.

I think the cheat sheet in the intermediate course covers this somewhat... What do you think @dc2917?

**Comment by dc2917** [comment]
Yeah, we have the cheat sheet/handout, and also the "Key Points" summary section at the end of the "[Committing and History](https://imperialcollegelondon.github.io/introductory_grad_school_git_course/l1-02-committing-history/index.html)", which lists the main git commands taught in the intro course.

**Comment by nicoloceneda** [comment]
Excellent, thanks!
